### ISSUE TITLE
Add new records

### ISSUE THREAD

**Issue by haesleinhuepf** [NFDI4BIOIMAGE/training#336]
This is a follow-up PR to #63 and related to #320.

We should add those resources, potentially with other links:

**NOT WORKING**

- https://www.fosteropenscience.eu/learning/use-open-data-in-teaching/#/id/5b06ca3ba8216e322da360b6
- https://gerbi-gmb.de/i3dbio/i3dbio-rdm/image-data-repositories/10.12688/f1000research.121714.2
- https://www.orca.nrw/searchresults?searchbox=%22elabftw%22&orca_show_all_of=result_media

**NO SECURE CONNECTION TO SITE POSSIBLE**

- https://dsw.fairdata.solutions/ 

**DID NOT FIND IT**

- Youtube channel Glencoe

related to #63

**Comment by haesleinhuepf** [comment]
@lea-33 would you mind taking care of these two, e.g. after #335 is merged?

**Comment by lea-33** [comment]
> @lea-33 would you mind taking care of these two, e.g. after #335 is merged?

Sure! I will solve it.
